Added connection timeout. Fixes #7417

sanderstad opened the pull request:
Added splatting fof command for readability

<!-- Below information IS REQUIRED with every PR -->
## Type of Change
<!-- What type of change does your code introduce -->
 - [X] Bug fix (non-breaking change, fixes #7417 )
 - [ ] New feature (non-breaking change, adds functionality, fixes #<!--issue number--> )
 - [ ] Breaking change (effects multiple commands or functionality, fixes #<!--issue number--> )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Unit test is included
 - [ ] Documentation
 - [ ] Build system

<!-- Below this line you can erase anything that is not applicable -->
### Purpose
The invoke command the connection timeout to make sure that commands would not execute for a long period of time

### Approach
Added variable to several query commands

### Commands to test
Invoke-DbaDbDataMasking

andreasjordan commented on functions/Invoke-DbaDbDataMasking.ps1:
```diff
@@ -367,7 +367,7 @@ function Invoke-DbaDbDataMasking {
                         $query = "CREATE NONCLUSTERED INDEX [$($maskingIndexName)] ON [$($dbTable.Schema)].[$($dbTable.Name)]([$($identityColumn)])"
 
-                        Invoke-DbaQuery -SqlInstance $server -SqlCredential $SqlCredential -Database $db.Name -Query $query
+                        Invoke-DbaQuery -SqlInstance $server -SqlCredential $SqlCredential -Database $db.Name -Query $query -QueryTimeout
```
Is there a ` $ConnectionTimeout` missing at the end?

andreasjordan replied:
Or better `$CommandTimeout`?

andreasjordan replied:
This is probably the reason the tests fail.

andreasjordan commented:
Why do you use `$ConnectionTimeout` for `-QueryTimeout` of `Invoke-DbaQuery`? You also have `$CommandTimeout`which you don not use at all and wich would be a better fit. And have a look at the documenation of the two parameters: They are the same.

sanderstad commented:
@andreasjordan You are correct, there was an obsolete parameter.
I removed the connectiontimeout parameter and replaced everything with the command timeout.
I'm not sure why I put that in, maybe to distinquish the connection to the servers and the query timeouts separately

potatoqualitee commented:
🙌🏼  awesome, thank you both!
